lib/js/src/manager/screen/_SoftButtonManagerBase.js

```diff
@@ -199,7 +199,9 @@ class _SoftButtonManagerBase extends _SubManagerBase {
                 }
                 if (initialState !== null && Array.isArray(softButtonObject.getStates())) {
                     for (const softButtonState of softButtonObject.getStates()) {
-                        if (softButtonState !== null && softButtonState.getName() !== null && this._sdlArtworkNeedsUpload(softButtonState.getArtwork())) {
+                        if (softButtonState !== null && softButtonState.getName() !== null && 
+                            this._fileManager !== null && this._fileManager !== undefined && 
```

Comment:
```suggestion
                        if (softButtonState !== null && softButtonState.getName() !== null &&
                            this._fileManager !== null && this._fileManager !== undefined &&
```
Fix issues found by linter. No trailing spaces allowed.